integration/tsconfig.json

```diff
@@ -0,0 +1,6 @@
+{
+  "compilerOptions": {
+    "types": ["cypress"]
+  },
+  "include": ["**/*.ts"]
+}
```

Comment:
sorry @ogbeche77 I mean you can remove the whole file

Reply:
>  "types": ["cypress"]

@pascal-airy  Oh, I see.I had intended to fix the `Cypress import error` currently shown by the editor. But I see we don't have that on develop

Reply:
ah cool. But it didn't work?